docs/operator-manual/user-management/google.md

```diff
@@ -182,6 +182,8 @@ Go through the same steps as in [OpenID Connect using Dex](#openid-connect-using
         apiVersion: v1
         kind: Secret
         metadata:
+          labels:
```

Comment:
Hi @EladLeev Can you please explain what was the impact of not having this before ?

Reply:
.